monkey/monkey_island/cc/server_utils/encryption/__init__.py

```diff
@@ -14,3 +14,5 @@
     unlock_datastore_encryptor,
     reset_datastore_encryptor,
 )
+from .encryption_key_types import EncryptionKey32Bytes
+from .errors import SizeError
```

Comment:
We can just use [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError).